### ISSUE TITLE
[BE][Active Sync - Admin, Editor, Viewer Access Control][9] As an admin, editor or viewer of an active synced collection, I should be able to perform these operations

### ISSUE THREAD

**Issue by nagesh953** [sparrowapp-dev/sparrow-api#147]
**Description**:
Respective Users should be able to view the elements/controls in the Active Sync flow as per the Admin, Editor or Viewer Access Control.

**Acceptance Criteria**:
Admin User:
Should be able to perform all the operations under the Active Sync.

Editor:
Should be able to perform all the operations under the Active Sync.

Viewer:
1. Should not be able to Import or create a collection. Should be able to view the already added collections or requests only.
2. Should not be able to manually sync the collection (Sync Collection button should be disabled).
3. Should not be able to add New requests.
4. Should not be able to delete the request.
5. Should not be able to restore the request.
6. Should not be able to switch the branches or delete the branches.
7. In right-click options at the collection or request level, should be able to open the collection or request only.
Basically a viewer should be able to send the API requests and should not be able to perform any post kind of operations.
